butterknife-sample-lib/build.gradle

```diff
@@ -0,0 +1,31 @@
+buildscript {
+  repositories {
+    mavenLocal()
+    mavenCentral()
+  }
+  dependencies {
+    classpath "${GROUP}:butterknife-gradle-plugin:${VERSION_NAME}"
+  }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'com.neenbedankt.android-apt'
+apply plugin: 'butterknife'
```

Comment:
It's unfortunate that you need both a plugin and the `@RClass` annotation. Couldn't this plugin modify the annotation processor options or register a task to generate a file with the right information?